SU2_CFD/src/fluid/CDataDrivenFluid.cpp

```diff
@@ -273,6 +339,9 @@ void CDataDrivenFluid::Run_Newton_Solver(su2double Y1_target, su2double Y2_targe
   /*--- 2D Newton solver, computing the density and internal energy values corresponding to Y1_target and Y2_target.
    * ---*/
 
+  AD::StartPreacc();
+  AD::SetPreaccIn(*Y1);
+  AD::SetPreaccIn(*Y2);
```

Comment:
Why are these variables passed by pointer if they are just scalars?

Reply:
Good point, I changed them to pass by reference.